test/unit/name_synchronizer_test.rb

```diff
@@ -37,8 +37,8 @@ def setup
     test "#sync_name synchronizes name based on interface" do
       refute_equal @host.name, @host.primary_interface.shortname
       @hsync.sync_name
-      assert_equal @host.name, @host.primary_interface.shortname
-      assert @host.name.blank?
+      assert_nil @host.name
+      assert_nil @host.primary_interface.shortname
     end
```

Comment:
This fixes: `DEPRECATED: Use assert_nil if expecting nil from test/unit/name_synchronizer_test.rb:40. This will fail in Minitest 6`